hibernate-core/src/main/java/org/hibernate/boot/spi/SessionFactoryOptions.java

```diff
@@ -308,4 +308,8 @@ default boolean areJPACallbacksEnabled() {
 		return true;
 	}
 
+	/**
+	 * Can bytecode-enhanced entity classes be used as a "proxy"?
+	 */
+	boolean isEnhancementAsProxyEnabled();
```

Comment:
Since this is a new SPI method, shouldn't this be added as a default method returning ```false```?

Reply:
Maybe